pages/linux/nmcli-device.md

```diff
@@ -0,0 +1,15 @@
+# nmcli device
+
+> Hardware device management with NetworkManager.
+
+- Print statuses of network interfaces:
+
+`nmcli device status`
+
+- Print the available Wi-Fi access points:
+
+`nmcli device wifi`
+
+- Connect to the Wi-Fi network with a specified name and password:
+
+`nmcli device wifi connect {{name}} password {{password}}`
```

Comment:
Perhaps we could tweak this slightly to be more intuitive?
```suggestion
`nmcli device wifi connect {{ssid}} password {{password}}`
```

Reply:
Done.